### PR TITLE
Cell watcher docstrings

### PR DESCRIPTION
**Pull request by bnmajor**

### CODE REVIEW

File: itkwidgets/cell_watcher.py [hunk 1]
```diff
@@ -1,69 +1,96 @@
...
-                names.append(name)
-        return names
+    def data(self) -> Dict[str, Dict[str, bool]]:
+        """Get the underlying data dict containg all viewer data
 
-    @property
-    def not_named(self):
-        # Return a list of names of viewers that have not been named yet
-        return any([k for k, v in self.data.items() if v['name'] is None])
+        :return: The data object that contains all created Viewer information.
```
Comment (thewtex) [inline itkwidgets/cell_watcher.py:26]: can we describe here what the `str`, `str`, `bool`'s are?

**Review by thewtex — CHANGES_REQUESTED** [review]
Wow, awesome!

One minor request inline.

File: itkwidgets/cell_watcher.py (hunk 1 above)
Comment (bnmajor) [reply itkwidgets/cell_watcher.py]: Sure thing!

**Review by thewtex — APPROVED** [review]
:100:

**Pull request merged**